projects/templates/src/lib/components/po-page-dynamic-detail/po-page-dynamic-detail.component.ts

```diff
@@ -313,4 +412,57 @@ export class PoPageDynamicDetailComponent implements OnInit {
     return !!value && typeof value === 'object' && !Array.isArray(value);
   }
 
+  private loadDataFromAPI() {
+    const { serviceApi, serviceMetadataApi, serviceLoadApi } = this.activatedRoute.snapshot.data;
+    const { id } = this.activatedRoute.snapshot.params;
+    const onLoad = serviceLoadApi || this.onLoad;
+    this.serviceApi = serviceApi || this.serviceApi;
+    this.poPageDynamicService.configServiceApi({ endpoint: this.serviceApi, metadata: serviceMetadataApi });
```

Comment:
Sugestão:
Adicionar algumas linhas em branco para melhorar a legibilidade. Para os demais desenvolvedores conseguirem identificar de forma rápida a implementação do método :)
```suggestion
  private loadDataFromAPI() {
    const { serviceApi: serviceApiFromRoute, serviceMetadataApi, serviceLoadApi } = this.activatedRoute.snapshot.data;
    const { id } = this.activatedRoute.snapshot.params;

    const onLoad = serviceLoadApi || this.onLoad;
    this.serviceApi = serviceApiFromRoute || this.serviceApi;

    this.poPageDynamicService.configServiceApi({ endpoint: this.serviceApi, metadata: serviceMetadataApi });

    const metadata$ = this.getMetadata(serviceApiFromRoute);
    const data$ = this.loadData(id);
    const customOption$ = this.loadOptionsOnInitialize(onLoad);

    this.subscriptions.push(concat(metadata$, data$, customOption$).subscribe());
  }    this.poPageDynamicService.configServiceApi({ endpoint: this.serviceApi, metadata: serviceMetadataApi });
```

Reply:
Também sugiro alterar a variavel de `serviceApi` para `serviceApiFromRoute`, ou algo semelhante, como no exemplo acima.

Reply:
Excelente sugestão! Aprendi  const { **serviceApi: serviceApiFromRoute**, serviceMetadataApi, serviceLoadApi } Obrigado!